go.mod

```diff
@@ -3,11 +3,32 @@ module github.com/CESARBR/knot-cloud-storage
 go 1.14
 
 require (
+	github.com/ahmetb/govvv v0.3.0 // indirect
+	github.com/cenkalti/backoff/v4 v4.0.0
+	github.com/cespare/reflex v0.2.0 // indirect
```

Comment:
Is it only my impression or a lot more changes are being added to this file? If that's the case, break into another commit.

Reply:
As we discussed, this changes were removed!